apps/site/components/withNavBar.tsx

```diff
@@ -20,12 +20,15 @@ import { useSiteNavigation } from '#site/hooks';
 import { useRouter, usePathname } from '#site/navigation.mjs';
 import { availableLocales } from '#site/next.locales.mjs';
 
-const SearchButton = dynamic(() => import('#site/components/Common/Search'), {
-  ssr: false,
-  loading: () => (
-    <Skeleton className={styles.searchButtonSkeleton} loading={true} />
-  ),
-});
+const SearchButton = dynamic(
```

Comment:
Do we still need it to be a dynamic component? At the very elast can we flip `ssr` to `true`?

Reply:
Since the new components has "use client" and assuming the components are compatible, we probably don't need the dynamic at all no?